src/index.html

```diff
@@ -7,11 +7,177 @@
       content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
     >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <title>HTML Form</title>
+    <title>Personal information</title>
     <link rel="stylesheet" href="./style.css">
+    <script type="text/javascript" src="./main.js"></script>
+
   </head>
   <body>
-    <h1>HTML Form</h1>
-    <script type="text/javascript" src="./main.js"></script>
+
+    <form>
+      <fieldset class="fieldset">
+        <legend>Personal information</legend>
+
+        <label for="surname" class="labelWithMargin">
+          Surname:
+          <input type="text"
+        name="usersurname"
+        autocomplete="off"
+          >
```

Comment:
```suggestion
          <input 
            type="text"
            name="userSurname"
            autocomplete="off"
          >
```
[Checklist](https://github.com/mate-academy/layout_html-form/blob/master/checklist.md) 3, 4 - fix formatting (everywhere)